pkgs/development/libraries/libcef/default.nix

```diff
@@ -0,0 +1,84 @@
+{ stdenv, mkChromiumDerivation, fetchgit, gtkglext
+, fontconfig, freetype, cairo, pango, libX11, libXi, libXext
+, libXfixes, libXrender, libXcomposite, dbus, atk, libXrandr
+, libXmu, libXt, libSM, libICE }:
+
+with stdenv.lib;
+
+mkChromiumDerivation (base: rec {
+  name = "libcef";
+  packageName = "cef";
+  # Run "git rev-list --count HEAD" to get this number
+  cefVersion = "1414";
+  # from CHROMIUM_BUILD_COMPATIBILITY.txt
+  upstreamVersion = "51.0.2704.47";
```

Comment:
I'm going to merge #15762 very soon, so I think this will break `libcef`. Can you please test building `libcef` after the merge is done? I'll Cc you in the merge commit so that you'll get notified.